contracts/nft-querier/Cargo.toml

```
@@ -0,0 +1,39 @@
+[package]
+authors = ["Vladislav Vasilev <vfaust.thefuture@gmail.com>"]
+description = "Contract to query NFT status"
```

For a better documentation sake, let's make it slightly more descriptive:

```suggestion
description = "Contract to query unbonding NFT status"
```